scripts/__tests__/changelog-generator.test.ts

```diff
@@ -70,13 +70,11 @@ describe(getOriginalCommit, () => {
   });
 });
 
-describe(getFirstCommitAfterForkingFromMaster, () => {
+describe(getFirstCommitAfterForkingFromMain, () => {
   it("returns the SHA of the first commit where its first parent is on the master branch", () => {
```

Comment:
Can you also fixup all refs to master while you're here? 😄 
```suggestion
  it("returns the SHA of the first commit where its first parent is on the main branch", () => {
```